ddp-workspace/projects/ddp-sdk/src/lib/services/logging.service.ts

```diff
@@ -22,11 +22,16 @@ export class LoggingService {
 
     public logError: Logger = this.showEvent(LogLevel.Error) ?
         (...args) => {
-            const stringifiedArgs = args.map(item => (
-                 (typeof item === 'object') ? this.stringify(item) : item
-            ));
-
-            this.stackdriverErrorReporterService.handleError(stringifiedArgs.join(', '));
+            const stringifiedArgs = args.map(arg => {
+                if (arg instanceof Error) {
+                    return arg.stack ? arg.stack : arg.message;
+                }
+                if (arg instanceof Object) {
+                    return this.stringify(arg);
+                }
+                return arg;
+            });
+            this.logToCloud(stringifiedArgs.join(',\n'), null, 'ERROR').pipe(take(1)).subscribe();
             console.error.apply(window.console, stringifiedArgs);
```

Comment:
Per discussion with Giorgi, call Google function `LoggingService` to write error to stackdriver log.

Also, modified how we handle `...args` to not lose any data because`stringify()` is not working for following case:

In the screenshot below, on the right side in debugger Scope, `stringifiedArgs` is the return value. It does not contains the the original error object anymore.

![logError](https://github.com/broadinstitute/ddp-angular/assets/35533885/b6f6ebeb-9108-4378-abbc-2f33d213ae57)